### PR TITLE
Updated Configuration Variables for DTE Energy Bridge

### PR DESCRIPTION
**Pull request by ChristofferG**
**Description:**
#6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/isy994.markdown [hunk 1]
```diff
@@ -30,15 +30,36 @@ isy994:
...
+password:
+  description: The password that used to access the ISY interface.
+  required: true
+  type: string
+sensor_string:
+  description: This is the string that is used to identify which devices are to be assumed to be sensors instead of lights of switches. By default, this string is 'sensor'. If this string is found in the device name or folder, Home Assistant will assume it is as a sensor or binary sensor (if the device has on/off or true/false states). This is only necessary for nodes that are not automatically detected as sensors by Home Assistant. Insteon door, window, motion and leak sensors should all be detected automatically.
+  required: false
+  type: string
+ignore_string:
+  description: Any devices that contain this string in their name (or folder path) will be ignored by Home Assistant. They will not become entities at all. The default value is `{IGNORE ME}`.
```
Comment (fabaff) [inline source/_components/isy994.markdown]: For the default should the `default:` key be used.

Comment (ChristofferG) [reply source/_components/isy994.markdown]: Ok thanks, I will add that.

**Comment by klaasnicolaas** [comment]
Is this going into the right branch? Normally I think I should go to the `current` branch

**Review by fabaff — APPROVED** [review]
Thanks :bird: 

I will cherry-pick it for `current`.

**Pull request merged**